### PR TITLE
d8 revision visibility

### PR DESCRIPTION
**Pull request by jkaeser**

### CODE REVIEW

**Comment by jessica-zivtech** [comment]
Tested this change in probo build https://240e8041-74cf-4d7d-9ef2-d1bf1a6289d7--pr-77.probo.build/

Recreation steps:
1. I created an Editor account. 
2. I verified that revision permissions were checked for the Editor role.
3. I logged in to the editor account and created a new basic page. The revision settings section was present, as expected.
4. I logged back into the admin account and unchecked the revision permissions for the Editor role.
5. I logged back into the editor account, created a new basic page, but was still seeing the revision settings section. I should not see this section after unchecking revision permissions. 

Note: I don't know if I needed to do this but I tried rebuilding the permissions after unchecking them as well but I was still seeing the revision settings section.

File: bear.profile [hunk 1]
```diff
@@ -45,4 +48,14 @@ function bear_form_node_form_alter(&$form, FormStateInterface $form_state) {
   if (isset($form['sticky'])) {
     $form['sticky']['#access'] = FALSE;
   }
+
+  $user = \Drupal::currentUser();
+  if (\Drupal::moduleHandler()->moduleExists('workbench_moderation') && isset($form['moderation_state'])) {
```
Comment (jodyHamilton) [inline bear.profile]: I dont even think workbench moderation has a d8 version, so you dont need to check for it

File: bear.profile [hunk 2]
```diff
@@ -45,4 +48,14 @@ function bear_form_node_form_alter(&$form, FormStateInterface $form_state) {
...
+    return;
+  }
+  if (!$user->hasPermission('view revisions')) {
```
Comment (jodyHamilton) [inline bear.profile]: \Drupal::currentUser()->hasPermission('name of permission'); is a little more concise than doing it in 2 lines

File: bear.profile (hunk 1 above)
Comment (jkaeser) [inline bear.profile]: It has an alpha version, but we haven't included it in bear yet. I'll remove this stuff for now.

**Comment by jessica-zivtech** [comment]
Re-tested in Probo. I was unable to see the revision settings section when I was logged in as an Editor who did not have the "View all revisions" permission.

**Comment by jkaeser** [comment]
Testing this on Probo wasn't working because I didn't change the value on line 17 in `build-a-bear.make` to `d8-revision-visibility`. I changed and pushed. @jessica-zivtech This is working now, correct?

**Comment by jkaeser** [comment]
@jodyHamilton I sent this for review to @jessica-zivtech, and she confirmed it's working so I changed the value on line 17 in `build-a-bear.make` back to `8.x`. This PR on to you for code review and merging!

**Pull request merged**